Source/Core/PolygonPipeline.js

```diff
@@ -41,12 +41,8 @@ define([
      */
     PolygonPipeline.computeArea2D = function(positions) {
         //>>includeStart('debug', pragmas.debug);
-        if (!defined(positions)) {
-            throw new DeveloperError('positions is required.');
-        }
-        if (positions.length < 3) {
-            throw new DeveloperError('At least three positions are required.');
-        }
+        Check.typeOf.object('positions', positions);
```

Comment:
looks like `positions` is just an array, so use `Check.defined('positions', positions)`